frontend/www/js/omegaup/problem_store.ts

```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+interface ProblemState {
+  showOverlay: boolean;
+}
+
+export const problemStore = new Vuex.Store<ProblemState>({
```

Comment:
```suggestion
export default const problemStore = new Vuex.Store<ProblemState>({
```

así se puede importar como

```typescript
import problemStore from 'frontend/www/js/omegaup/arena/ProblemStore';
```

Reply:
Creeeo que este `Store` ya no será necesario por los cambios que hicimos en los componentes `XxxPopup`, donde el encargado de mostrar/ocultar formularios es el componente padre. 

De caulquier modo lo voy a quitar de aquí y lo voy a agregar en el PR #4941. Si se decide quitar, que sea en el otro.